docs/guidelines/marketing_guide_to_firefox_l10n.md

```diff
@@ -0,0 +1,164 @@
+# Firefox localization copy guide
+
+This document is intended to facilitate the creation and localization of copy, and to support optimal preparation and consistency. This is a living document and it reflects our current state. As our brand expression continues to evolve, we&#39;ll make updates to this doc.
+
+**Why is this guide important for the localization of Firefox marketing copy?**
```

Comment:
We should use titles for this, instead of bold.

Reply:
Done